### PR TITLE
Add cheap smoothing to brush pointer

### PR DESCRIPTION
**Pull request by setphen**
Nothing fancy. Just a simple idea to create a smoother brush stroke. There are some edge cases where this might look a bit weird or sharp, still.

### CODE REVIEW

File: scripts/modules/brush.pointer.js [hunk 1]
```diff
@@ -19,24 +19,49 @@ function Pointer(offset = new Position(), color = new Color('000000'))
 
   this.draw = function()
   {
-    if(!this.position_prev){this.position_prev = this.position(); }
+    if(!this.position_prev){this.position_prev = [this.position()]; return; }
```
Comment (setphen) [inline scripts/modules/brush.pointer.js]: Added a return here to avoid drawing a line from the start position to itself.

**Comment by ragekit** [comment]
could you use and offset canvas and css blur, then use that as the image to draw ?

**Comment by neauoire** [comment]
This works surprisingly well actually.

**Comment by ragekit** [comment]
nice

**Pull request merged**